### ISSUE TITLE
Missing dependencies for Living Hinge (Fit to rectangle)

### ISSUE THREAD

**Issue by ChristopheMineau** [eridur-de/mightyscape-1.2#129]
Hi folks, just discovered your work, that's really great ! Thanks very much for putting together all this stuff !
I came here initially for the Living Hinge extension, I was desperate to find something working on Inkscape 1.2 and here you did the job,but, almost ...
I had at the first run the following errors 
![image](https://github.com/eridur-de/mightyscape-1.2/assets/23100833/5a959382-6baf-4fe3-980b-8fa675e83418)

So in total, I had to install additionally :
$ pip install lxml
$ pip install cssselect
$ pip install numpy

And it ended up working.
I am on W10, IS 1.2.2, I installed using the git clone method / pip install  ..... requirement.txt on my local python 3.10

Thanks again for the great work !

**Comment by vmario89** [comment]
Hi. Glad you got it running :) Looking into` https://github.com/eridur-de/mightyscape-1.2/blob/master/requirements.txt` this should be fine. lxml, csssselect, numpy are included in the requirements already. The error you have might belong to a wrong setting of the python interpreter. This can be configured in` ~/.config/inkscape/preferences.xml`

```
  <group
     id="extensions"
     python-interpreter="/usr/bin/python"
```

**Comment by ChristopheMineau** [comment]
Hi thanks for answering.
Well my python configuration is exactly as you suggest.
I did install in my python console using pip on your requirement file, but don't know why, these didn't get installed.
I know that it ended up failing on openmesh, maybe it's because of that ?

Anyway, now I am having a look at Boxes.py, and it also fails because it misses some dependencies, and these ones are not in the requirement.txt file :
I had to do :
pip install shapely
pip install affine
to make it work.

And, I know this starts to get away from the topic, but when boxes.py ends up working, I get drawings way out of scale.
For instance, I want a 100mm wide box, and I get with something like 10000mmx15000mm
I must be doing something wrong but what ?
Thanks again for your help
